### PR TITLE
Rewrote the tour page

### PR DESCRIPTION
**Pull request by casruger**
Rewrote the tour page and removed 'research tour' button from the home page under the search bar. However, could not find out how to remove the researcher tour page, that still has to get done.

### CODE REVIEW

**Comment by hannesdatta** [comment]
Hi @casruger, before merging, can you please make the changes as requested? I tracked my changes.

[tour_updated.docx](https://github.com/tilburgsciencehub/website/files/8619110/tour_updated.docx)

Could you please also involve @srosh2000 in the reformulation of the tour page? She had revised our last tour pages, and it may be useful to learn from her experience. 

In particular, I love the elements of @srosh2000 "old" tour to come back in the new tour. In particular, the short & catchy language, and the slogans we used ("e.g., code like a pro").

Please also discuss on how one tour page can onboard both students and researchers. I think the updated tour from @casruger makes it more apparent that our "old" vision (researchers) doesn't include students a lot...

Before launching, I think it makes sense you both agree on the new tour page. It will be a fruitful collaboration.
 
Thank you!

**Comment by hannesdatta** [comment]
I have revised the wording of the tour page here and there.

I feel the building block section is still a bit wordy - we could aim to reduce it more. But - for now, it's good to go? Can you please track the use of the search on the tour page to decide whether we want to keep it or not?

**Comment by srosh2000** [comment]
@hannesdatta @casruger ,
I just made a pull request with some minor changes to the tour page to make it more crisp. Overall, I like the additional "what we do", "what we offer" and "contribute" ( sense of community) aspect of the new tour. However, the part where we introduce tutorials and building blocks is still a bit too generic and I don't see it add too much value as you land on the same page by clicking on the upper tabs of tutorials/building blocks (next to the search bar). Moreover, you could simply hyper link the list of tutorials and building blocks pages under "what we offer" maybe?

 I especially liked the "narrative" aspect of the old tours which would indicate the purpose behind the building blocks etc that are linked. 

I'm thinking if there is a way to still incorporate the 2 separate tours with the narrative or include the narrative somehow in the single tour page version.
